Resources/en.lproj/Localizable.strings

```diff
@@ -301,6 +301,11 @@
 "PROTOCOL_NOT_SELECTED" = "Protocol not selected";
 "Settings" = "Settings";
 
+//Donation
+"DONATE" = "Donate to VLC";
+"SUPPORT_VLC" = "Support the volunteers building VLC";
+"DONATION_AMOUNT" = "Donation Amount";
+
```

Comment:
Do we want to be more descriptive here about how the money is being used?

Reply:
since this is just on top of a button we don't have a lot of space.